ext/pdo_firebird/tests/fb4_session_timezone.phpt

```diff
@@ -0,0 +1,59 @@
+--TEST--
+PDO_Firebird: Firebird 4.0 set session time zone
+--EXTENSIONS--
+pdo_firebird
+--SKIPIF--
+<?php require('skipif.inc'); 
+if (Pdo\Firebird::getApiVersion() < 40) {
+	die('skip: Firebird API version must be greater than or equal to 40');
+}
+require 'testdb.inc';
+$dbh = getDbConnection();
+$stmt = $dbh->query("SELECT RDB\$get_context('SYSTEM', 'ENGINE_VERSION') AS VERSION FROM RDB\$DATABASE");
+$data = $stmt->fetch(\PDO::FETCH_ASSOC);
+if (!$data || !array_key_exists('VERSION', $data) || version_compare($data['VERSION'], '4.0.0') < 0) {
+	die("skip Firebird Server version must be greater than or equal to 4.0.0");
+}
```

Comment:
I think we should introduce a respective function in testdb.inc, to avoid having to write (or copy/paste) this code again and again (we're using it already in fb_datatypes.phpt and fb_datatypes_params.phpt). Maybe the signature of that function could be `function checkMinServerVersion(string $version): void`.

Reply:
Makes sense. Perhaps we should also improve the `getDbConnection` function so that it can pass a parameter with additional connection properties (`TRANSACTION_ISOLATION_LEVEL`, `SESSION_TIMEZONE`).

I have a patch template with another connection attribute `BIND_DATATYPES`.

Reply:
```php
function connectToDb(array $params = []): Pdo\Firebird {
    return Pdo\Firebird::connect(PDO_FIREBIRD_TEST_DSN, PDO_FIREBIRD_TEST_USER, PDO_FIREBIRD_TEST_PASS, $params);
}

function checkMinServerVersion(string $version): void
{
    $dbh = connectToDb();
    $stmt = $dbh->query("SELECT RDB\$get_context('SYSTEM', 'ENGINE_VERSION') AS VERSION FROM RDB\$DATABASE");
    $data = $stmt->fetch(\PDO::FETCH_ASSOC);
    if (!$data || !array_key_exists('VERSION', $data) || version_compare($data['VERSION'], $version) < 0) {
	die("skip Firebird Server version must be greater than or equal to version");
}
}
```

Reply:
These are good additions. I suggest to add this to this PR.

Reply:
Done